c2mir/c2mir.c

```diff
@@ -10469,8 +10491,10 @@ static MIR_label_t get_label (c2m_ctx_t c2m_ctx, node_t target) {
   return labels->attr = MIR_new_label (c2m_ctx->ctx);
 }
 
+static op_t top_gen_last_op;
```

Comment:
This a trick I made to get result of `N_EXPR` operation, as it call `top_gen`, this operation should be exactly the one the `N_STMTEXPR` should return.